react-native-purchases-ui/src/index.tsx

```diff
@@ -18,11 +28,36 @@ export function presentPaywallIfNeeded(requiredEntitlementIdentifier: String) {
   RNPaywalls.presentPaywallIfNeeded(requiredEntitlementIdentifier);
 }
 
-const ComponentName = 'Paywall';
-
-export const Paywall =
-  UIManager.getViewManagerConfig(ComponentName) != null
-    ? requireNativeComponent(ComponentName)
+const InternalPaywall =
+  UIManager.getViewManagerConfig('Paywall') != null
+    ? requireNativeComponent('Paywall')
     : () => {
       throw new Error(LINKING_ERROR);
     };
+
+export const Paywall: React.FC<FlexPaywallFooterViewProps> = ({style, children}) => (
+  <InternalPaywall style={[{flex: 1}, style]}/>
+);
+
+const InternalPaywallFooterView = UIManager.getViewManagerConfig('Paywall') != null
+  ? requireNativeComponent('RCPaywallFooterView')
+  : () => {
+    throw new Error(LINKING_ERROR);
+  };
+
+type FlexPaywallFooterViewProps = {
+  style?: StyleProp<ViewStyle>;
+  children?: ReactNode;
+};
+
+export const PaywallFooterContainerView: React.FC<FlexPaywallFooterViewProps> = ({style, children}) => (
+  <View style={[{flex: 1}, style]}>
+    <ScrollView
+      style={{backgroundColor: '#4b72f6'}}
```

Comment:
Is this on purpose? Should this be transparent?

Reply:
I left it like this to be able to see the scrollview and helped me see a bunch of issues, like the loading issues I mention in the PR description, but blue is kinda intense, I will change it with a light grey